src/api/window/window.ts

```diff
@@ -536,12 +536,17 @@ export class _Window extends Base {
     }
 
     /**
-     * Closes the window
-     * @param { boolean } interrupt assigns the value to false
-     * @return {Animation}
+     * Performs the specified window transitions.
+     * @param {object} transitions - Describes the animations to preform. See the tutorial.
```

Comment:
typo: preform -> perform
transitions type {object} needs to update to 'Transition'